Feat: add menu to header (#507)
okp4/ui: stories/molecules/components/header/Header.stories.mdx

  </Story>
</Canvas>

## Navigation menu

> The optional `navigationMenu` is used to hightlight the main navigation of your website .  

Reviewer: Typo 😆
```suggestion
The optional `navigationMenu` is used to hightlight the main navigation of your website.  
```

Author: Eagle eye 🦅 !

> This responsive menu makes available a set of navigable links to the user and greatly improves its experience by providing several easily accessible shortcuts.  

Reviewer: Feel free to space out the paragraphs to make it more readable. It's up to you.

> This responsive menu makes available a set of navigable links to the user and greatly improves its experience by providing several easily accessible shortcuts.  

Reviewer: ```suggestion
This responsive menu provides the user with a set of navigable links and greatly improves his experience by providing several easily accessible shortcuts.
```

You just have to provide it with an array of `JSX.Element` which will then be displayed on a single line on a large or medium screen, or accessible via a burger menu on a smaller or mobile screen.  

> Please shorten the window to let the magic happen!

Reviewer: 😍

<Canvas>
  <Story name="Navigation menu">
    {() => {
      const changeHeaderDisplay = useMediaType('(max-width: 995px)')
      const links = [
        { name: 'Whitepaper', url: 'https://docs.okp4.network/docs/whitepaper/abstract' },
        { name: 'Røadmap', url: 'https://okp4.network/#roadmap' },

>         { name: 'Faucet', url: 'https://faucet.devnet.staging.okp4.network/' },

Reviewer 2: That's a great idea to use real links but we must be careful as these links could change in a near future (ex: Faucet will come soon in TestNet) or could be broken.
Thus, I think it's better for now to use either fake links or something like GitHub ones (or stuff like that).

Author: yes you're right.
